tests/test_hdfs.py

```diff
@@ -229,8 +231,9 @@ def has_running_executors():
                 "--conf", "spark.cores.max=8",
                 "--conf", "spark.executors.cores=4"]
 
+    data_dir = "hdfs:///users/alice"
     driver_id = utils.submit_job(app_url=utils.SPARK_EXAMPLES,
-                                 app_args="10.0.0.1 9090 hdfs:///users/alice/netcheck hdfs:///users/alice/outfile",
+                                 app_args="10.0.0.1 9090 {dir}/netcheck {dir}/outfile".format(dir=data_dir),
```

Comment:
Out of scope of this PR, but we could use the same (or similar) wrapper `_run_terasort_job` here.